modulefiles/tasks/derecho/run_vx.local.lua

```diff
@@ -1,6 +1,25 @@
 --[[
 Compiler-specific modules are used for met and metplus libraries
 --]]
-load(pathJoin("met", os.getenv("met_ver") or "10.1.2"))
-load(pathJoin("metplus", os.getenv("metplus_ver") or "4.1.3"))
+local met_ver = (os.getenv("met_ver") or "10.1.2")
+local metplus_ver = (os.getenv("metplus_ver") or "4.1.3")
+if (mode() == "load") then
+  load(pathJoin("met", met_ver))
+  load(pathJoin("metplus",metplus_ver))
+end
+local base_met = os.getenv("met_ROOT") or os.getenv("MET_ROOT")
+local base_metplus = os.getenv("metplus_ROOT") or os.getenv("METPLUS_ROOT")
+
+setenv("MET_INSTALL_DIR", base_met)
+setenv("MET_BIN_EXEC",    pathJoin(base_met,"bin"))
+setenv("MET_BASE",        pathJoin(base_met,"share/met"))
+setenv("MET_VERSION",     met_ver)
+setenv("MET_VERSION",     metplus_ver)
```

Comment:
@RatkoVasic-NOAA - please change line 17 to set "METPLUS_VERSION" instead of "MET_VERSION" (my mistake; it is similar in other run_vx.local.lua files as well):
`setenv("MET_VERSION", metplus_ver)`